src/manifest.json

```
@@ -412,6 +396,32 @@
           }
         }
       ]
+    },
+    {
+      "type": "group",
+      "id": "vss",
+      "label": "Vector similarity search explained",
```

We don't use the term "vector similarity" anymore; just "vector search".

```suggestion
      "label": "Vector search explained",
```